universal-login-react/src/ui/WalletSelector/Suggestions/Suggestion.tsx

```diff
@@ -11,7 +11,15 @@ interface SuggestionWithTypeProps extends SuggestionProps {
 export const Suggestion = ({type, ...props}: SuggestionWithTypeProps) => {
   switch (type) {
     case 'SingleCreation':
-      return (<SingleCreationSuggestion {...props} />);
+      return (<SingleSuggestion
+        hint='This username is available'
+        {...props}
+      />);
+    case 'SingleConnection':
+      return (<SingleSuggestion
+        hint='Do you want connect to this account?'
```

Comment:
`Do you want to connect to this account?`

Reply:
👍  ✔️